fixtures/rules/config.json

```diff
@@ -0,0 +1,10 @@
+{
+  "enabled": true,
+  "channel": "beta",
+  "config": {
+    "rules": "java-basic,java-design"
```

Comment:
Is there a reason why this isn't an array (instead of a string with a comma-separated list)?

Reply:
Because I am stupid I guess... =/

Reply:
Ha, no way! 😛 

Just a suggestion to simplify parsing. 😄

Reply:
Actually we just forward the entire thing, I think that was the reason I didn't bother to split, but I makes more sense from a `yml` perspective to have it as an array.

Reply:
@toddmazierski Adjusted.